scripts/build-tag-push-ecr.sh

```diff
@@ -4,16 +4,15 @@ set -eu
 
 echo "building image(s)"
 
-cd "${WORKING_DIRECTORY}"
-echo "Packaging app in /$WORKING_DIRECTORY"
+echo "Packaging sam app in /$WORKING_DIRECTORY"
```

Comment:
```suggestion
echo "Packaging sam app in $WORKING_DIRECTORY"
```
This throw me off before